pages/common/black.md

```diff
@@ -0,0 +1,24 @@
+# black
+
+> A speedy Python auto code formatter.
```

Comment:
I would say we remove the word "speedy" as we try to omit any adjectives from the command description. Writing with adjectives almost feels like an ad.
```suggestion
> A Python auto code formatter.
```